### ISSUE TITLE
Remove "multiple contours" warning when segmentation masks contains holes

### ISSUE THREAD

**Issue by Beno71** [SchmollerLab/Cell_ACDC#139]
I was just analyzing a position where a  cell contained holes in the segmentation mask in multiple frames.
In that case, ACDC displays a warning "Current frame contains cells with multiple contours" where entering the next frame needs to be explicitly confirmed by pressing "Yes".

Should we differentiate between holes and the real case of multiple (distinct) contours? I often saw single pixels in YeaZ predictions within cells which were classified as background and therefore produced an additional contour within a cell. In those cases it would be more convenient to have an additional warning "holes in cells" which does not trigger a dialogue where users have to explicitly press a key to continue.

**Comment by ElpadoCan** [comment]
I agree we should differentiate between multiple objects with same ID vs holes

**Comment by ElpadoCan** [comment]
Related issue #140

**Comment by ElpadoCan** [comment]
Done. I removed the objects with holes from the multiple contours warning. Objects with holes now are notified only in the title with red text and do not trigger a dialogue.
